media/css/firefox/tiles.less

```diff
@@ -109,7 +109,17 @@
 }
 
 @media only screen and (min-width: @breakTablet) and (max-width: @breakDesktop) {
-
+ 
+    #main-feature h1{
```

Comment:
Coding style fix - please make sure there is a space after the selector and before the opening `{`. (Here and in `.main-column` below.)